docusaurus.config.js

```diff
@@ -59,25 +59,11 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: "Home",
+        title: "Developer",
```

Comment:
IMO this title should probably reflect the domain we choose, but either way this should be plural:

```suggestion
        title: "Developers",
```